### PR TITLE
fix CMAKE warning about un-unit-testable CPP file in physics unit tests

### PR DESCRIPTION
**Pull request by AndrewMeadows**
* remove cmake warning about un-unit-testable CPP file
* fix some broken phyics unit tests

### CODE REVIEW

**Comment by AndrewMeadows** [comment]
This PR fixes a CMAKE warning about a CPP file not having the correct format for unit testing.  Every couple of weeks a new developer would ask me about this warning so I decided to fix it.

There is nothing QA testable here, except by building the code and noticing that this warning does not show up anymore:

```
CMake Warning at cmake/macros/SetupHifiTestCase.cmake:72 (message):
  ignoring .cpp file (not a test class -- this will not be linked or
  compiled!): /home/andrew/work/wtf/tests/physics/src/MeshUtil.cpp
```

Also, if you build the physics-test target on linux it actually builds again.  It was broken accidentally a couple of months ago when I switched the `ShapeManager` over to hand out **const** pointers rather than just **non-const** ones.

File: tests/physics/src/MeshUtil.cpp [hunk 1]
```diff
@@ -1,45 +0,0 @@
...
-    }
-    // scan for outside edge
-    for (auto& edgeEntry : edges) {
-        if (edgeEntry.second == 1) {
-             return false;
-        }
-    }
-    return true;
-}
-
```
Comment (AndrewMeadows) [inline tests/physics/src/MeshUtil.cpp]: This code moved into the header to eliminate the CMAKE warning.  It does not yet make sense for this code to live in a library -- it is purely for some physics unit tests.

File: tests/physics/src/ShapeManagerTests.cpp [hunk 2]
```diff
@@ -219,14 +219,14 @@ void ShapeManagerTests::addCompoundShape() {
...
-    btCollisionShape* shape = shapeManager.getShape(info);
+    const btCollisionShape* shape = shapeManager.getShape(info);
     QVERIFY(shape != nullptr);
 
     // verify the shape is correct type
     QCOMPARE(shape->getShapeType(), (int)COMPOUND_SHAPE_PROXYTYPE);
 
     // verify the shape has correct number of children
-    btCompoundShape* compoundShape = static_cast<btCompoundShape*>(shape);
+    const btCompoundShape* compoundShape = static_cast<const btCompoundShape*>(shape);
```
Comment (AndrewMeadows) [inline tests/physics/src/ShapeManagerTests.cpp]: These changes to `const` pointers fix the **physics-test** build target.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7642e9fd1e29726f0062407b0c4b8876ee237c65 available at: https://gist.github.com/23de8804552c12cf979c9304dda37587<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8740-7642e9f.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8740-7642e9f.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e46c15401aec77c88d746923787bd709394ef85e available at: https://gist.github.com/23de8804552c12cf979c9304dda37587<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8740-e46c154.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8740-e46c154.dmg

**Comment by Atlante45** [comment]
Ran unit tests, all tests passed.

**Pull request merged**